docs/nightly/en/user-guide/manage-data/overview.md

```diff
@@ -0,0 +1,196 @@
+
+For example, you can create an `app_logs` table with the `append_mode` option as follows.
+The `host` and `log_level` columns represent tags, and the `ts` column represents the time index.
+
+```sql
+CREATE TABLE app_logs (
+    ts TIMESTAMP TIME INDEX,
+    host STRING,
+    api_path STRING FULLTEXT,
+    log_level STRING,
+    log STRING FULLTEXT,
+    PRIMARY KEY (host, log_level)
+) WITH ('append_mode'='true');
+```
+
+Insert a new row into the `app_logs` table:
+
+```sql
+INSERT INTO app_logs (ts, host, api_path, log_level, log)
+VALUES ('2024-07-11 20:00:10', 'host1', '/api/v1/resource', 'ERROR', 'Connection timeout');
+```
+
+Check the data in the table:
+
+```sql
+SELECT * FROM app_logs;
+```
+
+The output will be:
+
+```sql
++---------------------+-------+------------------+-----------+--------------------+
+| ts                  | host  | api_path         | log_level | log                |
++---------------------+-------+------------------+-----------+--------------------+
+| 2024-07-11 20:00:10 | host1 | /api/v1/resource | ERROR     | Connection timeout |
++---------------------+-------+------------------+-----------+--------------------+
+1 row in set (0.01 sec)
+```
+
+You can insert new data with the same tag and time index:
+
+```sql
+INSERT INTO app_logs (ts, host, api_path, log_level, log)
+-- The same tag `host1` and `ERROR`, the same time index 2024-07-11 20:00:10
+VALUES ('2024-07-11 20:00:10', 'host1', '/api/v1/resource', 'ERROR', 'Connection reset');
+```
+
+Then you will find two rows in the table:
+
+```sql
+SELECT * FROM app_logs;
+```
+
+```sql
++---------------------+-------+------------------+-----------+--------------------+
+| ts                  | host  | api_path         | log_level | log                |
++---------------------+-------+------------------+-----------+--------------------+
+| 2024-07-11 20:00:10 | host1 | /api/v1/resource | ERROR     | Connection reset   |
+| 2024-07-11 20:00:10 | host1 | /api/v1/resource | ERROR     | Connection timeout |
++---------------------+-------+------------------+-----------+--------------------+
+2 rows in set (0.01 sec)
+```
+
+## Delete Data
+
+You can effectively delete data by specifying tags and time index.
+Deleting data without specifying the tag and time index columns is not efficient, as it requires two steps: querying the data and then deleting it by tag and time index.
+For more information about column types, please refer to the [Data Model](../concepts/data-model.md).
+
+:::warning Warning
+Excessive deletions can negatively impact query performance.
+:::
+
+You can only delete data using SQL.
+For example, to delete a row from the `monitor` table with tag `host` and timestamp index `ts`:
+
+```sql
+DELETE FROM monitor WHERE host='127.0.0.2' AND ts=1667446798450;
+```
+
+The output will be:
+
+```sql
+Query OK, 1 row affected (0.00 sec)
+```
+
+For more information about the `DELETE` statement, please refer to the [SQL DELETE](/reference/sql/delete.md).
```

Comment:
Add a new section to describe `truncata table`, which delete all table data in an efficient way.

For example `truncate table monitor;`

I'll add the `truncate` reference document.